vault/diagnose/tls_verification.go

```diff
@@ -122,3 +123,30 @@ func TLSFileChecks(certFilePath, keyFilePath string) error {
 
 	return nil
 }
+
+// TLSClientCAFileCheck checks whether client CA file is readable and parsable or not
+func TLSClientCAFileCheck(l *configutil.Listener) error {
+
+	caPool := x509.NewCertPool()
+	data, err := ioutil.ReadFile(l.TLSClientCAFile)
```

Comment:
This works, but I think for the sake of reusing code, we should use https://github.com/hashicorp/vault/blob/13629ca2d4e8ddd221086751d9a09d75bd46cf49/vault/diagnose/tls_verification.go#L97